modules/data-loading/pages/data-streaming-connector.adoc

```diff
@@ -56,12 +56,14 @@ The value for this configuration should be `com.google.cloud.hadoop.fs.gcs.Googl
 The streaming connector supports the following file types:
 
 * CSV files
+* JSON files.
+Each JSON object must be on a separate line.
 * directories
 * tar files
 * zip files
 
-For URIs that point to directories and compressed files, the connector will look for all CSV files inside the directory or the compressed file.
-If you set `file.recursive`, the connector will look for CSV files recursively.
+For URIs that point to directories and compressed files, the connector looks for all CSV or JSON files inside the directory or the compressed file.
```

Comment:
Not quite. It will try to load everything if file.regexp=".*", if file.regexp='.json', it will load json ONLY. You can also do regex expressions like ".(json|txt)" which will load json AND txt files.

Reply:
Thanks for clarifying! I've clarified in the document